### ISSUE TITLE
Postop examples with NEON target

### ISSUE THREAD

**Issue by aobolensk** [ARM-software/ComputeLibrary#1032]
Hi,
I have a question about postops on NEON. I have seen functional examples when postops are used with CL target.
Also, GEMMInfo in CpuGemmConv2d operator seems to have support for postops.
But is there any example and use case of using postops specifically with NEON (not CL)?

**Comment by morgolock** [comment]
Hi @aobolensk 

Could you please point us to the code you are referring to? What kind of post operation would you like to compute?

In general ACL does not support dynamic fusion of operators in the neon backend.

Hope this helps.

**Comment by aobolensk** [comment]
Hi @morgolock 
I mean using PostOp classes defined in arm_compute/core/experimental/PostOps.h applied to NEON layers. I have found that OpenCL convolution and GEMM variant has an option to use postops for activation and elemwise add operation. I have tried to add postops in cl_gemm_reshaped_rhs_only.cpp sample and postop is executed. But I have not found any examples of using postops for NEON target convolution anf GEMM.
Are postops supported for NEON target? If so, are there any examples of using that?
